NAMESPACE

```diff
@@ -108,6 +108,7 @@ importFrom(rgeos,gIntersects)
 importFrom(rgeos,gLength)
 importFrom(rgeos,gSimplify)
 importFrom(stats,setNames)
+importFrom(utils,URLencode)
```

Comment:
Heads-up: I think that when you import it, you can use the bare `URLencode` function without the `::`. 

See here: http://r-pkgs.had.co.nz/namespace.html#imports

> It’s common for packages to be listed in Imports in DESCRIPTION, but not in NAMESPACE. In fact, this is what I recommend: list the package in DESCRIPTION so that it’s installed, then always refer to it explicitly with pkg::fun().